docker-compose.deploy.yml

```diff
@@ -35,12 +35,7 @@ services:
         condition: service_healthy
     command: ["sh", "-c", "pnpm exec prisma migrate deploy && node /usr/src/app/aio_run.mjs"]
     healthcheck:
-      test:
-        - CMD
-        - curl
-        - '-f'
-        - 'http://localhost:80'
-      interval: 2s
-      timeout: 10s
-      retries: 30
-
+  test: [ "CMD", "curl", "-f", "-s", "-o", "/dev/null", "-w", "%{http_code}", "http://localhost:80" ]
+  interval: 2s
+  timeout: 10s
+  retries: 30
```

Comment:
@progprnv Could you fix the line indentation and rebase with the latest changes?